packages/rocketchat-videobridge/client/public/external_api.js

```diff
@@ -0,0 +1,377 @@
+/**
+ * Implements API class that embeds Jitsi Meet in external applications.
+ */
+var JitsiMeetExternalAPI = (function()
+{
+    /**
+     * The minimum width for the Jitsi Meet frame
+     * @type {number}
+     */
+    var MIN_WIDTH = 790;
+
+    /**
+     * The minimum height for the Jitsi Meet frame
+     * @type {number}
+     */
+    var MIN_HEIGHT = 300;
+
+    /**
+     * Constructs new API instance. Creates iframe element that loads
+     * Jitsi Meet.
+     * @param domain the domain name of the server that hosts the conference
+     * @param room_name the name of the room to join
+     * @param width width of the iframe
+     * @param height height of the iframe
+     * @param parent_node the node that will contain the iframe
+     * @param filmStripOnly if the value is true only the small videos will be
+     * visible.
+     * @constructor
+     */
+    function JitsiMeetExternalAPI(domain, room_name, width, height, parentNode,
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (17)](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2736174701/issues/source?bid=3199634&fileBranchId=3342079#l30)